### DISCUSSION TITLE
通过应用商店安装的jumpserver4.2.0无法通过admin/ChangeMe登录

### DISCUSSION THREAD

**Discussion by mr-nealon** [1Panel-dev/1Panel#7056] [Q&A]
### 1Panel 版本

1.1.20

### 问题描述

通过应用商店安装的jumpserver4.2.0无法通过admin/ChangeMe登录

### 重现步骤

通过应用商店安装的jumpserver4.2.0无法通过admin/ChangeMe登录

**Comment by wanghe-fit2cloud** [comment]
可以先进入 `容器` 页面，查看 JumpServer 容器的运行状态和容器日志。

**Comment by wanghe-fit2cloud** [comment]
刚刚在本地环境测试，应用运行正常且可访问。

**Reply by mr-nealon** [reply]
![image](https://github.com/user-attachments/assets/28ffd8b8-0235-4149-8068-7d391b03bf53)

**Reply by mr-nealon** [reply]
不行

**Reply by wanghe-fit2cloud** [reply]
在浏览器中通过 `admin/ChangeMe` 登录之后，再来看下容器日志中是否有 Error 相关内容。
> 也可以将该问题提交给 JumpServer Issue。
